packages/shared/src/components/cards/SourceButton.tsx

```diff
@@ -1,13 +1,12 @@
 import React, { CSSProperties, ReactElement } from 'react';
 import { TooltipPosition } from '../tooltips/BaseTooltipContainer';
 import { LinkWithTooltip } from '../tooltips/LinkWithTooltip';
-import { getSourcePermalink, Source } from '../../graphql/sources';
```

Comment:
is this function `getSourcePermalink` used anywhere else?
Else we can clean it up perhaps.

Reply:
Right, let's clean all implementations.